packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -35,18 +35,26 @@ async function initDefaultWorkflow({ workflowsService, stagesService, strapi })
 
 function extendReviewWorkflowContentTypes({ strapi }) {
   const extendContentType = (contentTypeUID) => {
-    const setStageAttribute = set(`attributes.${ENTITY_STAGE_ATTRIBUTE}`, {
-      writable: true,
-      private: false,
-      configurable: false,
-      visible: false,
-      useJoinTable: true, // We want a join table to persist data when downgrading to CE
-      type: 'relation',
-      relation: 'oneToOne',
-      target: 'admin::workflow-stage',
-    });
-    strapi.container.get('content-types').extend(contentTypeUID, setStageAttribute);
+    const setAttribute = (path, target) =>
+      set(path, {
+        writable: true,
+        private: false,
+        configurable: false,
+        visible: false,
+        useJoinTable: true, // We want a join table to persist data when downgrading to CE
+        type: 'relation',
+        relation: 'oneToOne',
+        target,
+      });
+
+    const setReviewWorkflowAttributes = pipe([
+      setAttribute(`attributes.${ENTITY_STAGE_ATTRIBUTE}`, 'admin::workflow-stage'),
+      setAttribute(`attributes.${ENTITY_ASSIGNEE_ATTRIBUTE}`, 'admin::user'),
+    ]);
```

Comment:
WDYT of renaming the function to be a bit more specific?

```suggestion
    const setRelation = (path, target) =>
      set(path, {
        writable: true,
        private: false,
        configurable: false,
        visible: false,
        useJoinTable: true, // We want a join table to persist data when downgrading to CE
        type: 'relation',
        relation: 'oneToOne',
        target,
      });

    const setReviewWorkflowAttributes = pipe([
      setRelation(`attributes.${ENTITY_STAGE_ATTRIBUTE}`, WORKFLOW_MODEL_UID),
      setRelation(`attributes.${ENTITY_ASSIGNEE_ATTRIBUTE}`, 'admin::user'),
    ]);
```